src/Sentry.Bindings.Cocoa/Sentry.Bindings.Cocoa.csproj

```diff
@@ -13,7 +13,7 @@
   </PropertyGroup>
 
   <!-- Build empty assemblies when not on macOS, to pass the solution build. -->
-  <ItemGroup Condition="!$([MSBuild]::IsOSPlatform('OSX'))">
+  <ItemGroup Condition="!($([MSBuild]::IsOSPlatform('OSX')) Or $([MSBuild]::IsOSPlatform('windows')))">
```

Comment:
I think this is going to be problematic in CI. With these files included, it's going to attempt to build the source for Sentry.Bindings.Cocoa on the Windows runner... and that will fail.

Rather than checking for the operating system as a second parameter, perhaps you could check for the existence of an environment variable (`EnableRemoteMacBuilds` or something) so that anyone developing on Windows with a remote Mac available could turn this on if they want, but it won't mess with the CI builds?

Reply:
That's a good idea!